sentinel/src/transitions/utils.js

```diff
@@ -41,11 +41,11 @@ module.exports = {
   addRegistrationNotFoundError: (doc, reportConfig) => {
     module.exports.addRejectionMessage(doc, reportConfig, 'registration_not_found');
   },
-  isIdUnique: (db, id, callback) => {
-    db.medic.view('medic-client', 'contacts_by_reference', {
+  isIdUnique: (id, callback) => {
+    db.medic.query('medic-client/contacts_by_reference', {
       key: [ 'shortcode', id ]
     }, (err, results) => {
-      callback(err, !!(results && results.rows && results.rows.length));
+      callback(err, !(results && results.rows && results.rows.length));
```

Comment:
You've reversed the logic here, but it looks like you've fixed it? How is it possible this was working before?

Reply:
It wasn't working before :(

The only place that this function is called is [here](https://github.com/medic/medic/blob/fdf98448ea84398a5ba4e40d531eb7ffc69352cb/sentinel/src/transitions/registration.js#L465) , where it expects it to return `true` when unique, `false` otherwise. 

If you want I can remove this, I found at least one more bug in Sentinel which I intend to fix separately, and I can include this as well.

Reply:
I think we should raise a new issue for this bug. If anyone was relying on this in production they will have to change their configuration and raising an issue will make sure it gets into the release notes.

The git history would probably be cleaner if this was fixed separately, but no worries if it's not worth the effort.

Reply:
I've reverted the fix, going to create a separate issue for it.